build-system/tasks/runtime-test.js

```diff
@@ -57,16 +57,33 @@ function getConfig() {
   return extend(obj, karmaConfig.default);
 }
 
-function getAdExtensions() {
-  const adNetworks = [];
+function getAdTypes() {
```

Comment:
looks good. Just one thing that confused me at first glance. I thought adType should have included all ad types. But then I realized it doesn't include `doubleclick` and `adsense`. Can we still call it `adExtensions`?